src/parser/mrt/messages/table_dump_v2_message.rs

```diff
@@ -182,22 +167,19 @@ pub fn parse_rib_entry(
     safi: &Safi,
     prefixes: &[NetworkPrefix],
 ) -> Result<RibEntry, ParserError> {
-    if input.remaining() < 8 {
-        // total length - current position less than 16 --
-        // meaning less than 16 bytes available to read
-        return Err(ParserError::TruncatedMsg("truncated msg".to_string()));
-    }
+    // total length - current position less than 16 --
+    // meaning less than 16 bytes available to read
+    input.require_n_remaining(8, "rib entry")?;
 
     let peer_index = input.read_u16()?;
     let originated_time = input.read_u32()?;
     if add_path {
+        // TODO: Why is this value unused?
```

Comment:
I don't exactly remember why it has path_id here, but the `read_nlri_prefix` function will read the `path_id` and override it anyway, so it is ignored here.